index.js

```diff
@@ -850,8 +857,14 @@ Command.prototype.description = function(str) {
  */
 
 Command.prototype.alias = function(alias) {
-  if (0 == arguments.length) return this._alias;
-  this._alias = alias;
+  var command = undefined;
+  if(this.commands.length == 0) {
+    command = this;
+  } else {
+    command = this.commands[this.commands.length - 1]
```

Comment:
I think we can remove `bin + 'l'` test  from `test.command.executableSubcommandAlias.js`.